recipes/AdobongIsda.js

```diff
@@ -37,7 +37,7 @@ render(){
           bottom: 0}}>
 
           <Text style={{margin:20,fontSize: 24, color:"black" }}>Adobong Isda</Text>
-          <Text style={{fontSize: 18, color:"black" }}>Ingredients</Text>
+          <Text style={{fontSize: 18, color:"black" }}>INGREDIENTS</Text>
 
           <Text style={{fontSize: 15, color:"black" }}>1/2 kilo Sapsap</Text>
```

Comment:
I-capitalize mo rin sila, iyan example. Pasensya na. Hahaha

To this:
1/2 Kilo Sapsap

*Basta capitalize yung first word

Reply:
ok. copy